packages/web/app/src/components/target/explorer/super-graph-metadata.tsx

```
@@ -49,6 +73,12 @@ function SubgraphChip(props: {
       content={
         <>
           <span className="font-bold">{props.text}</span> subgraph
+          {props.metadata?.map(m => (
+            <>
+              <br />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)